test/integration/test_resolvers.py

```diff
@@ -0,0 +1,24 @@
+"""Integration tests for the conda dependency resolution."""
+
+import os
+from base import integration_util
+from base.api import ApiTestCase
+
+GNUPLOT = {u'version': u'4.6', u'type': u'package', u'name': u'gnuplot'}
+
+class CondaResolutionIntegrationTestCase(integration_util.IntegrationTestCase, ApiTestCase):
```

Comment:
No - that is what you should be doing.

I think it breaks at this commit - https://bitbucket.org/mozman/svgwrite/commits/e661623be032d6060cf87f4f9df1bbf3b0d3011f when they make pyparsing and external dependency.